### DISCUSSION TITLE
Hide / Disable Environment for improved Transparency

### DISCUSSION THREAD

**Discussion by Jojo-banana** [kinsi55/CS_BeatSaber_Camera2#47] [Q&A] [answered]
It would be great to add the possibility to disable the environment from a camera, this would help a lot with transparent cameras because the background environment often gets in the way.

**Comment by kinsi55** [comment, marked as answer]
Removing the entire Environment (And just that) is not easy - Not something I would consider adding. As mentioned in the processing tab, the intended way to get around this is to set a [shorter render](https://github.com/kinsi55/CS_BeatSaber_Camera2/wiki/Camera-Configuration#farz) distance for the given camera

**Reply by kinsi55** [reply]
In the latest version I've added a Visibility option that covers "Everything else" which is not configurable specifically - The environment would also fall in that category
